pre/main.ipynb

```diff
@@ -141,8 +136,20 @@
    "outputs": [],
    "source": [
     "@step\n",
-    "def parse_json(event):\n",
-    "    return json.loads(event)"
+    "def check_secret(event):\n",
```

Comment:
OK, but I'd prefer to have a new web source built into bspump which checks the secret and doesn't even emit an event if the secret is invalid. This seems to be a pattern that we are repeating.

Reply:
OK, I'll look at it